### DISCUSSION TITLE
Versione di Home Assistant

### DISCUSSION THREAD

**Discussion by mippo9** [indomus/forum#2719] [HUB: Home Assistant]
Scusate la domanda banale, ma da dove si capisce qual'è la versione di HA che sta effettivamente girando ?
Cioè io credo di stare aggiornando HA tutti i mesi, seguendo la procedura per Core, Docker QNAP, ma da un po' di tempo sotto Impostazioni, Informazioni, ci trovo:
" Home Assistant 2023.3.1 Frontend 20230302.0 - latest "

Quindi è possibile che sia la versione di marzo 2023 ?
In effetti più o meno da luglio, durante la re-installazione mensile, non mi propone più la scelta tre le varie versioni, quella dove dovevo indicare "Stable".
Cosa vuol dire, che sto continuando a installare la stessa versione del marzo 2023 ?!?

**Comment by 50494554524F** [comment]
si, la versione installata è 2023.3.1

**Comment by mippo9** [comment]
ok, grazie, purtroppo è quello che temevo ...

Come si fa ad ottenere l'immagine della versione stabile per QNAP Container Station ? 
Fino ad ora, io indicavo:
Registry:  -->  Docker Hub
Image:  -->  homeassistant/home-assistant

non dava errore ed apparentemente installava tutto correttamente ...

**Comment by mippo9** [comment]
Ho modificato "Image", scrivendo "homeassistant/home-assistant:stable"

ora Impostazioni, Informazioni mi da:
Core 2023.11.2
Frontend 20231030.2

che mi sembra molto più corretto di prima ...!
(Graziemille)
